specification/keyvault/resource-manager/Microsoft.KeyVault/preview/2020-04-01-preview/managedHsm.json

```diff
@@ -0,0 +1,624 @@
+          "type": "string",
+          "description": "The name of the managed HSM Pool."
+        },
+        "type": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The resource type of the managed HSM Pool."
+        },
+        "location": {
+          "type": "string",
+          "description": "The supported Azure location where the managed HSM Pool should be created.",
+          "x-ms-mutability": [
+            "create",
+            "read"
+          ]
+        },
+        "sku": {
+          "$ref": "#/definitions/ManagedHsmSku",
+          "description": "SKU details"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags"
+        }
+      },
+      "description": "Managed HSM resource",
+      "x-ms-azure-resource": true
+    },
+    "ManagedHsmListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ManagedHsm"
+          },
+          "description": "The list of managed HSM Pools."
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to get the next set of managed HSM Pools."
+        }
+      },
+      "description": "List of managed HSM Pools"
+    },
+    "ManagedHsmError": {
+      "properties": {
+        "error": {
+          "readOnly": true,
+          "description": "The server error.",
+          "$ref": "#/definitions/Error"
+        }
+      },
+      "description": "The error exception."
+    },
+    "Error": {
+      "properties": {
+        "code": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The error code."
+        },
+        "message": {
+          "type": "string",
+          "readOnly": true,
+          "description": "The error message."
+        },
+        "innererror": {
+          "x-ms-client-name": "innerError",
+          "readOnly": true,
+          "description": "The inner error, contains a more specific error code.",
+          "$ref": "#/definitions/Error"
+        }
+      },
+      "description": "The server error."
+    }
+  },
+  "parameters": {
+    "SubscriptionIdParameter": {
+      "name": "subscriptionId",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Subscription credentials which uniquely identify Microsoft Azure subscription. The subscription ID forms part of the URI for every service call."
+    },
+    "ApiVersionParameter": {
+      "name": "api-version",
+      "in": "query",
+      "required": true,
+      "type": "string",
+      "description": "Client Api Version."
+    },
+    "ResourceGroupName": {
+      "name": "resourceGroupName",
+      "in": "path",
+      "required": true,
+      "type": "string",
+      "description": "Name of the resource group that contains the key vault.",
```

Comment:
key vault => managed hsm

Reply:
This is an issue in Open API validation tools which forces to use same description for this property "resourceGroupName".
https://teams.microsoft.com/l/message/19:f1b9607b948840cca8b7852ed6bb1713@thread.skype/1586458312040?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=a5e9b41d-1b0a-471d-a170-8ce202fa30b1&parentMessageId=1586458312040&teamName=OpenAPI%20Hub%20Support&channelName=General&createdTime=1586458312040

Reply:
Fixed this as well.